extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClientConfig.java

```diff
@@ -190,4 +190,28 @@ public interface MongoClientConfig {
      */
     Optional<UuidRepresentation> uuidRepresentation();
 
+    enum TransportConfig {
+        /**
+         * Uses a Netty-based transport re-using the existing Netty event loops.
+         */
+        NETTY,
+        /**
+         * With a reactive driver it uses an async transport backed by a driver-managed thread pool,
+         * while with a blocking driver it uses a blocking transport.
```

Comment:
The non reactive one should just run inlined in the worker pool we provide - this by checking the mongo Java driver code + debugging it